src/modules/esl-event-listener/core/targets/inert.target.ts

```diff
@@ -0,0 +1,101 @@
+import {SyntheticEventTarget} from '../../../esl-utils/dom/events/target';
+import {resolveDomTarget} from '../../../esl-utils/abstract/dom-target';
+import {isElement} from '../../../esl-utils/dom/api';
+import {bind} from '../../../esl-utils/decorators/bind';
+import {resolveCSSSize} from '../../../esl-utils/dom/units';
+import {aggregate} from '../../../esl-utils/async/aggregate';
+import {ESLEventListener} from '../listener';
+
+import {ESLWheelEvent} from './inert.target.event';
+
+import type {ESLWheelEventInfo} from './inert.target.event';
+import type {CSSSize} from '../../../esl-utils/dom/units';
+import type {ESLDomElementTarget} from '../../../esl-utils/abstract/dom-target';
+
+export {ESLWheelEvent};
+
+/**
+ * Describes settings object that could be passed to {@link ESLWheelTarget.for} as optional parameter
+ */
+export interface ESLWheelSetting {
+  /** The minimum distance to accept as a long scroll (supports `px`, `vw`, and `vh` units) */
+  threshold?: CSSSize;
+  /** The maximum duration of the wheel events to consider it inertial */
+  timeout?: number;
+}
+
+/**
+ * Implementation of EventTarget to observe wheel events for inertial scrolling
+ */
+export class ESLWheelTarget extends SyntheticEventTarget {
+  protected static defaultConfig: Required<ESLWheelSetting> = {
+    threshold: '400px',
```

Comment:
Firsts of all, I don't think we are interested in size units here
And the most important thing, I don't think we handle delta properties of the WheelEvent correct
https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent/deltaY

Reply:
Do we also need to track z-axis delta?